code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -340,16 +340,32 @@
 	if(hasHUD(user,"security"))
 		var/perpname = get_visible_name(TRUE)
 		var/criminal = "None"
+		var/commentLatest = "None"
 
 		if(perpname)
 			for(var/datum/data/record/E in data_core.general)
 				if(E.fields["name"] == perpname)
 					for(var/datum/data/record/R in data_core.security)
 						if(R.fields["id"] == E.fields["id"])
 							criminal = R.fields["criminal"]
+			if(commentLatest)
+				for(var/datum/data/record/E in data_core.general)
+					if(E.fields["name"] == perpname)
+						for(var/datum/data/record/R in data_core.security)
+							if(R.fields["id"] == E.fields["id"])
+								if(LAZYLEN(R.fields["comments"]))

```

Comment:
Put this in the loop done just above. It also searches the perp in the db and then assigns the criminal status. Best to search the last comment status in the same loop

Reply:
Done!